web/concrete/js/build/core/app/edit-mode/editmode.js

```diff
@@ -574,13 +574,7 @@
                 pos = $container.offset(),
                 l = pos.left;
 
-            var tw = l + parseInt($toolbar.width());
```

Comment:
Are we sure we want to undo this? This leads to this on a desktop browser:

![screen shot 2015-02-16 at 1 37 46 pm](https://cloud.githubusercontent.com/assets/527809/6219626/09e40c42-b5e1-11e4-8152-aaf9a073d0cc.png)

There's got to be a way to undo this just for small screens.